src/controller/InvokeInteraction.h

```diff
@@ -75,7 +75,14 @@ InvokeCommandRequest(Messaging::ExchangeManager * aExchangeMgr, const SessionHan
     VerifyOrReturnError(commandSender != nullptr, CHIP_ERROR_NO_MEMORY);
 
     ReturnErrorOnFailure(commandSender->AddRequestData(commandPath, requestCommandData, timedInvokeTimeoutMs));
-    ReturnErrorOnFailure(commandSender->SendCommandRequest(sessionHandle));
+    if (responseTimeout.HasValue())
+    {
+        ReturnErrorOnFailure(commandSender->SendCommandRequest(sessionHandle, responseTimeout.Value()));
```

Comment:
Seems like we should just have SendCommandRequest() take an Optional....

Reply:
If you're cool with me changing the API, that would definitely be better. Doing it. THanks.

Reply:
Yes, that API is so not sacred.  :)  Change it, please!

Reply:
done.